wandb/sdk/artifacts/artifact.py

```diff
@@ -261,9 +261,23 @@ def _from_name(
         if is_artifact_registry_project(project):
             try:
                 entity = InternalApi()._resolve_org_entity_name(entity, organization)
-            except ValueError as e:
-                wandb.termerror(str(e))
-                raise
+            except ValueError as entity_error:
```

Comment:
Are you trying to check for `org_error` only in the event of an `entity_error`?

Reply:
first try to fetch org entity with entity if that fails and there is an org, then try to fetch org entity with the org that the user passed in just in case it was an org entity.

For the error i merged both together so if both fail it looks something like this:
```
wandb: ERROR Error resolving organization of entity: 'estellazx', failed with error: ValueError("Unable to resolve an organization associated with the entity: 'estellazx' that is initialized in the API or Run settings. This could be because 'estellazx' is a personal entity or the team entity doesn't exist. Please re-initialize the API or Run with a team entity using wandb.Api(overrides={'entity': '<my_team_entity>'}) or wandb.init(entity='<my_team_entity>') ").
wandb: ERROR Defaulted to trying to use 'potato' as an org entity and failed with error: ValueError("Unable to resolve an organization associated with the entity: 'potato' that is initialized in the API or Run settings. This could be because 'potato' is a personal entity or the team entity doesn't exist. Please re-initialize the API or Run with a team entity using wandb.Api(overrides={'entity': '<my_team_entity>'}) or wandb.init(entity='<my_team_entity>') ").
```

Not the most elegant but does the job in my opinion.

Reply:
Ah i get it now